package.json

```diff
@@ -24,7 +26,7 @@
   },
   "homepage": "https://zen-audio-player.github.io",
   "devDependencies": {
-    "html5-lint": "^0.2.5",
+    "htmlhint": "latest",
```

Comment:
Can we at least stick to the `0.9.x` line? I don't want something crazy to happen if they go to `1.0.0`